assembl/static2/tests/unit/components/harvesting/__snapshots__/taxonomyOverflowMenu.spec.jsx.snap

```diff
@@ -6,326 +6,354 @@ exports[`taxonomyOverflowMenu component should match harvestingMenu snapshot 1`]
       >
-        <Translate
-          value="search.taxonomy_action.mix_match"
+        <Unknown
+          backgroundColor="#fff"
+          color="#000"
+          qualifier="mix_match"
         />
+        <div
+          className="action-label"
+        >
+          <Translate
+            value="search.taxonomy_action.mix_match"
+          />
+        </div>
       </div>
-    </div>
-    <div
-      className="taxonomy-label"
-      onClick={[Function]}
-    >
-      <Unknown
-        backgroundColor="#fff"
-        color="#000"
-        qualifier="display_multi_column"
-      />
       <div
-        className="action-label"
+        className="taxonomy-label"
+        onClick={[Function]}
       >
-        <Translate
-          value="search.taxonomy_action.display_multi_column"
+        <Unknown
+          backgroundColor="#fff"
+          color="#000"
+          qualifier="display_multi_column"
         />
+        <div
+          className="action-label"
+        >
+          <Translate
+            value="search.taxonomy_action.display_multi_column"
+          />
+        </div>
       </div>
-    </div>
-    <div
-      className="taxonomy-label"
-      onClick={[Function]}
-    >
-      <Unknown
-        backgroundColor="#fff"
-        color="#000"
-        qualifier="display_thread"
-      />
       <div
-        className="action-label"
+        className="taxonomy-label"
+        onClick={[Function]}
       >
-        <Translate
-          value="search.taxonomy_action.display_thread"
+        <Unknown
+          backgroundColor="#fff"
+          color="#000"
+          qualifier="display_thread"
         />
+        <div
+          className="action-label"
+        >
+          <Translate
+            value="search.taxonomy_action.display_thread"
+          />
+        </div>
       </div>
-    </div>
-    <div
-      className="taxonomy-label"
-      onClick={[Function]}
-    >
-      <Unknown
-        backgroundColor="#fff"
-        color="#000"
-        qualifier="display_tokens"
-      />
       <div
-        className="action-label"
+        className="taxonomy-label"
+        onClick={[Function]}
       >
-        <Translate
-          value="search.taxonomy_action.display_tokens"
+        <Unknown
+          backgroundColor="#fff"
+          color="#000"
+          qualifier="display_tokens"
         />
+        <div
+          className="action-label"
+        >
+          <Translate
+            value="search.taxonomy_action.display_tokens"
+          />
+        </div>
       </div>
-    </div>
-    <div
-      className="taxonomy-label"
-      onClick={[Function]}
-    >
-      <Unknown
-        backgroundColor="#fff"
-        color="#000"
-        qualifier="display_open_questions"
-      />
       <div
-        className="action-label"
+        className="taxonomy-label"
+        onClick={[Function]}
       >
-        <Translate
-          value="search.taxonomy_action.display_open_questions"
+        <Unknown
```

Comment:
Do you know what are these `Unknown` in the snapshot (I saw some `undefined` also, just ensure those are "normal")

Reply:
I think it's because we do a shallow rendering, these are the sub components.